src/parser.cpp

```diff
@@ -479,6 +479,168 @@ options_metadata parser::load_settings()
         "node.refresh_transactions",
         value<bool>(&configured.node.refresh_transactions),
         "Request transactions on each channel start, defaults to false."
+    )
+
+    /* [bitcoin] */
+    (
+        "bitcoin.retargeting_factor",
+        value<uint32_t>(&configured.bitcoin.retargeting_factor),
+        "The retargeting factor, defaults to 4."
+    )
+    (
+        "bitcoin.target_spacing_seconds",
+        value<uint32_t>(&configured.bitcoin.target_spacing_seconds),
+        "The target block spacing time in seconds, defaults to 600."
+    )
+    (
+        "bitcoin.easy_spacing_seconds",
+        value<uint32_t>(&configured.bitcoin.easy_spacing_seconds),
+        "The easy target block spacing time in seconds, defaults to 1200."
+    )
+    (
+        "bitcoin.timestamp_future_seconds",
+        value<uint32_t>(&configured.bitcoin.timestamp_future_seconds),
+        "The amount of seconds the timestamp is allowed to drift, defaults to 7200."
+    )
+    (
+        "bitcoin.target_timespan_seconds",
+        value<uint32_t>(&configured.bitcoin.target_timespan_seconds),
+        "The target timespan for the retargeting interval in seconds, defaults to 1209600."
+    )
+    (
+        "bitcoin.proof_of_work_limit",
+        value<uint32_t>(&configured.bitcoin.proof_of_work_limit),
+        "The proof of work limit, defaults to 486604799."
+    )
+    (
+        "bitcoin.min_timespan",
+        value<uint32_t>(&configured.bitcoin.min_timespan),
+        "The lower bound for the retargeting timespan, defaults to 302400."
+    )
+    (
+        "bitcoin.max_timespan",
+        value<uint32_t>(&configured.bitcoin.max_timespan),
+        "The upper bound for the retargeting timespan, defaults to 4838400."
+    )
+    (
+        "bitcoin.retargeting_interval",
+        value<size_t>(&configured.bitcoin.retargeting_interval),
+        "The retargeting inverval in amount of blocks, defaults to 2016."
+    )
+    (
+        "bitcoin.genesis_block",
+        value<config::block>(&configured.bitcoin.genesis_block),
+        "The genesis block in hex, defaults to Satoshi's genesis block."
```

Comment:
Don't abbreviate hex, describe encodings, refer to Satoshi or redundantly to genesis. Recommend:
```
The genesis block.
```

Reply:
Done.